Dockerfile

```diff
@@ -105,7 +110,7 @@ RUN pip install -e . --no-deps
 FROM backend as prod
 
 # Copy frontend build over
-COPY --from=frontend /fides/clients/admin-ui/out/ /fides/src/fides/ui-build/static/admin
+COPY --from=built_frontend /fides/clients/admin-ui/out/ /fides/src/fides/ui-build/static/admin
```

Comment:
this isn't a required change for this PR, but I noticed it and thought it might be an easy win. hopefully I didn't break anything!

Reply:
looks like an easy win to me 🤔